python/py-macfsevents/Portfile

```diff
@@ -4,12 +4,12 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-macfsevents
+python.rootname     MacFSEvents
 version             0.8.1
 revision            0
-platforms           darwin
 license             BSD
 
-python.versions     27 35 36 37 38 39
+python.versions     27 37 38 39
```

Comment:
Is there any reason to keep py27-macfsevents subport?

Reply:
```
% port echo depends:py27-macfsevents
ninja-ide
```